leadbutt.py

```diff
@@ -173,50 +351,90 @@ def get_metric_statistics(**kwargs):
     if 'aws_secret_access_key' in auth_options:
         connect_args['aws_secret_access_key'] = auth_options['aws_secret_access_key']
     conn = boto.ec2.cloudwatch.connect_to_region(region, **connect_args)
-    for metric in config['Metrics']:
-        options = get_options(
-            config_options, metric.get('Options'), cli_options)
+    if 'Metrics' in config:
```

Comment:
Let's do the same here as `enhanced_monitoring`. Do `config.get('Metrics', False)`.